src/dmd/backend/cod3.d

```diff
@@ -4095,8 +4095,11 @@ void prolog_loadparams(ref CodeBuilder cdb, tym_t tyf, bool pushalloc, out regm_
         {
             shadowregm |= mask(preg);
             opcode_t op = 0x89;                  // MOV x[EBP],preg
+            type *ta = t;
```

Comment:
`type* ta = t`

`*` goes next to the type

Reply:
alright